bin/manage

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+docker compose exec -it dashboard pipenv run python manage.py $@
```

Comment:
As for other utility scripts, don't forget to define the appropriate user:

```suggestion
#!/usr/bin/env bash

set -eo pipefail

declare DOCKER_USER
DOCKER_UID="$(id -u)"
DOCKER_GID="$(id -g)"
DOCKER_USER="${DOCKER_UID}:${DOCKER_GID}"

DOCKER_USER=${DOCKER_USER} \
  DOCKER_UID=${DOCKER_UID} \
  DOCKER_GID=${DOCKER_GID} \
  docker compose exec -it \
    dashboard \
    pipenv run python manage.py "$@"
```

Also, the `set -eo pipefail` is essential for a shell script! For reference, see: https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425